src/main/java/net/dries007/tfc/CommonEventHandler.java

```diff
@@ -195,7 +195,7 @@ public static void onBlockHarvestDrops(BlockEvent.HarvestDropsEvent event)
         }
 
         // Drop shards from glass
-        if (state.getMaterial() == Material.GLASS && Constants.RNG.nextInt(2) == 0)
+        if (state.getMaterial() == Material.GLASS && Constants.RNG.nextInt(2) == 0 && block != Blocks.GLASS_PANE)
```

Comment:
I think we could instead make the number of dropped glass shard items be proportional to the broken block, ie:
```
Glass Blocks = 1-4
Glass Pane = 0-1
```
And then, either a mold, device or something to smelt those glass shards back into blocks or panes, not only blocks.

Lastly, if this idea is not agreed upon, better to just check for `instanceof BlockGlass` or `instanceof BlockStainedGlass` otherwise, your code will still drop shards for stained glass panes and other modded glass panes.

Reply:
From the issue, I believe the current conversion is 1 shard = 1 glass block. How about 50% chance of dropping a shard from a block, and 25% chance from anything else? 
I'd rather not implement a new device or mechanic to close this dupe bug in 1.12.

Reply:
No `instanceof`. No material or block equality check. Use an ore dict entry, this is a perfect example for it.

Reply:
Oh yeah, I didn't consider ore dicts, they are much better for this purpose. Still, Gael's solution would make the weird scenario where a glass pane drops a shard that can be smelt into a block (where the mass for this block came?). So, for the time being better to use Alc's idea on ore dicts and just check for `blockGlass` to drop a single shard. In the future, if we do implement a glass mechanic in base TFC this could be revisited.

Reply:
Oh, good. I'll do this then.

Reply:
Well. I tried. See the new commit. Getting an itemStack of a random block that broke in the world isn't straightforward, I think? getPickBlock wants nonNull target and player. ItemAnimalHide#onItemUse is the only place I saw that does something similar, but it already knows the player, and passes a null target. Could make it so glass only drop shards if an entity breaks them?

Reply:
Ok, just looked up Block#getItem and did that instead. Think this is ready.